client/connector/sdc/create_test.go

```diff
@@ -119,7 +120,11 @@ func TestCreate(t *testing.T) {
 
 			testCase.setupFunc()
 
-			output, err := sdc.Create(context.Background(), *http.MustNewWithDefault(baseUrl, "a_valid_token"), *sdc.NewCreateInput(testCase.sdcName))
+			output, err := sdc.Create(
+				context.Background(),
+				*http.MustNewWithConfig(baseUrl, "a_valid_token", 0, 0, time.Minute),
+				*sdc.NewCreateInput(testCase.sdcName),
```

Comment:
Create client with no retries and delay, so that the unit tests do not need to wait for the usual retries and delay.